### ISSUE TITLE
sklearn.metrics arguments

### ISSUE THREAD

**Issue by laurenjli** [Computational-Content-Analysis-2020/frequently-asked-questions#11]
Hi,

I noticed something in the order of the sklearn.metrics arguments and wanted to clarify. Should the order be predictions, true labels or the other way around? 

In the jupyter notebook, predictions come first: sklearn.metrics.precision_score(test_data_df['nb_predict'], test_data_df['category'])

But in the documentaiton, it says that arguments should be sklearn.metrics.precision_score(y_true, y_pred)

Thanks!

**Comment by bhargavvader** [comment]
Uh oh - you're absolutely right. Changing this right away.
